acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature

```diff
@@ -98,16 +98,16 @@ Feature: mod-consortia integration tests
     * call read('order-utils/organizations.feature')
     * call read('order-utils/orders.feature')
 
-  Scenario: Create and open order
+  Scenario: Open order with locations from different tenants
     Given call read('features/open-order-with-locations-from-different-tenants.feature')
 
   Scenario: Reopen order and change instance connection orderLine
     Given call read('features/reopen-and-change-instance-connection-order-with-locations-from-different-tenants.feature')
 
-  Scenario: Test cross-tenant inventory objects creation when working with pieces
```

Comment:
The reason changed title, is to be more clear in cucumber report which scenario belong to which feature